src/slurm_plugin/clustermgtd.py

```diff
@@ -74,10 +142,20 @@ def __repr__(self):
         attrs = ", ".join(["{key}={value}".format(key=key, value=repr(value)) for key, value in self.__dict__.items()])
         return "{class_name}({attrs})".format(class_name=self.__class__.__name__, attrs=attrs)
 
+    def __eq__(self, other):
+        if type(other) is type(self):
+            return self._config == other._config
```

Comment:
nit: Does comparing 2 ConfigParser object "just works"? I could not find the documentation, please double check

Reply:
I did test it manually but let me add a unit test for it: https://github.com/aws/aws-parallelcluster-node/pull/229